ssh.go

```diff
@@ -41,7 +45,8 @@ var (
 				Render
 )
 
-func sshServer(ln net.Listener) {
+func SshServer(ln net.Listener, vm string) {
```

Comment:
Capitalization of common initialisms
```suggestion
func SSHServer(ln net.Listener, vm string) {
```